NewForce-Cohort-4/react-nutshell-puffinor
# Articles

## What?
Created pages for articles for the user to be able to add a new article, see the time it was posted, a summary, and the date it was posted on. the article name should be a hyperlink where user can visit the article site.  

## Why?
Worked on meeting customer requirments in Trello. 

## How?
All code is commented and used JS react to code.

## Testing?
- Add Article feature
- Ability to enter information in text fields
- Delete an article
- Click on name hyperlink 
- See time it was posted 

## Anything Else?
I hope it works!
